src/lib.rs

```diff
@@ -63,6 +63,9 @@ pub mod nats;
 #[cfg(feature = "neo4j")]
 #[cfg_attr(docsrs, doc(cfg(feature = "neo4j")))]
 pub mod neo4j;
+#[cfg(feature = "oracle")]
+#[cfg_attr(docsrs, doc(cfg(feature = "oracle")))]
+pub mod oracle;
```

Comment:
Probably it makes sense to name the module as `oracle_free` or something like that. In order to highlight that.
It's pretty common pattern as far as I can see, e.g:
- Testcontainers for Java: https://java.testcontainers.org/modules/databases/oraclefree/
- Testcontainers for Python: https://github.com/testcontainers/testcontainers-python/tree/main/modules/oracle-free
- and couple more not related to testcontainers at all

But the same time we can keep the feature as `oracle`, because we may expose other images later (e.g XE) 🤔